src/sources/http_client/client.rs

```diff
@@ -285,7 +285,7 @@ impl HttpClientContext {
     }
 
     /// Enriches events with source_type, timestamp
-    fn enrich_events(&self, events: &mut Vec<Event>) {
+    pub fn enrich_events(&self, events: &mut Vec<Event>) {
```

Comment:
this comment is obsolete since below I recommend getting rid of this function in this impl , but for posterity I am not sure why the function needed to be made pub.